extension/src/experiments/columns/collect/index.ts

```diff
@@ -19,6 +20,7 @@ const collectFromExperiment = (
 ) => {
   const { data } = experiment
   if (data) {
+    collectTimestamp(acc)
```

Comment:
[I] If you look at the rest of this file we are now collecting a static column (constant) for every experiment and every branch. We don't need to do that. We can simply create the accumulator with the constant included.